policy/modules/admin/usermanage.te

```diff
@@ -500,6 +501,8 @@ init_dontaudit_rw_utmp(sysadm_passwd_t)
 
 logging_send_syslog_msg(sysadm_passwd_t)
 
+userdom_manage_admin_dirs(sysadm_passwd_t)
```

Comment:
Thinkig about it again, I'd rather not have it here just because an editor wants to update a config file.